src/components/NavBar/formatTimestamp.tsx

```diff
@@ -0,0 +1,28 @@
+const formatTimestamp = (entry: any) => {
```

Comment:
Let's not use `any` here, instead let's use `Post`

using `any` takes away from Typescript

```suggestion
const formatTimestamp = (entry: Post) => {
```

Reply:
Nice suggestion. I also don't think the type should be a `Post` either. Posts aren't the only place where we display timestamps, so I think we should pass in the timestamp itself rather then the whole post. In this case the type would be a `string`